just/bluefin-apps.just

```diff
@@ -0,0 +1,156 @@
+# vim: set ft=make :
+########################
+### bluefin-apps.just
+########################
+## Standardized verbs
+# configure- = configure something that is pre-installed on the image
+# install-   = install something, no uninstall or configuration provided
+# setup-     = install something and also provide configuration and/or uninstallation options
+# toggle-    = turn something on/off, logic can be automatic or manual selection
+# fix-       = apply fix/patch/workaround for something
+# foo        = no verb is used for shortcuts or something deemed important enough to use a super memorable name
+
+alias brew := install-brew
+
+# Install Homebrew | https://brew.sh
+install-brew:
+    #!/usr/bin/env bash
+    source /usr/lib/ujust/ujust.sh
+    if [[ ! -f "/var/home/linuxbrew/.linuxbrew/bin" || ! -x "/var/home/linuxbrew/.linuxbrew/bin/brew" ]]; then
+        echo "${b}Brew Installation${n}"
+        echo "Please ${b}IGNORE${n} everything the installer tells you to do at the end"
+        echo "We have already done it for you! You just need to close and re-open the terminal after installation"
+        echo "Do you understand?"
+        echo "Please type in \"YES I UNDERSTAND\" and press enter"
+        read ACCEPT
+        if [ "$ACCEPT" == "YES I UNDERSTAND" ]; then
+          /usr/bin/bash -c '/usr/libexec/brew-install'
+        else
+          echo "Capitalization matters when you type \"YES I UNDERSTAND\""
+        fi
+    fi
+    # if /etc/profile.d/brew.sh already exists, replace it with /usr/etc/profile.d/brew.sh
+    if [ -f /etc/profile.d/brew.sh ]; then
+        if [ -f /usr/etc/profile.d/brew.sh ]; then
+            sudo cp /usr/etc/profile.d/brew.sh /etc/profile.d/brew.sh
+        fi
+    fi
```

Comment:
I get that we may want to do this, but instead of copying over, maybe a symlink might make more sense? This might be a change we need to do on our /etc/profile.d/brew.sh

If that's a file then the user has modified, if it's a symlink we won't have to update? Since this prompt doesn't go away, I would want to avoid overwriting user changes.

Reply:
Can change that, i think @bketelsen wrote that originally, might be a reason why it is not a symlink instead.
Will wait for them to reply before making that change though, but i am pretty sure a symlink will be fine.

Reply:
no reason, symlink is a great idea. please proceed!